packages/teleport/src/Player/DesktopPlayer.tsx

```diff
@@ -65,7 +77,39 @@ export const ProgressBarDesktop = (props: { playerClient: PlayerClient }) => {
       });
     });
 
+    const throttledUpdateCurrentTime = throttle(
+      currentTimeMs => {
+        setState(prevState => {
+          return {
+            ...prevState,
+            current: currentTimeMs,
+            time: toHuman(currentTimeMs),
+          };
+        });
+      },
+      // Magic number to throttle progress bar updates so that the playback is smoother.
+      50
```

Comment:
What do you think about bumping it to 60ms? once per frame?

Reply:
Correct me if I'm wrong, but I don't think the frame rate is that straight forward in the browser (hence the need for [`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)).

Reply:
You're right, ignore me 😄

Reply:
Well maybe it would be smoother if we synchronized it using `requestAnimationFrame`.

Reply:
Updated this issue to include that suggestion: https://github.com/gravitational/webapps/issues/579